packages/sdk/package.json

```diff
@@ -14,7 +14,7 @@
     "start": "tsdx watch",
     "build": "tsdx build",
     "test": "tsdx test",
-    "test:e2e": "tsdx test --config jest-e2e-config.json",
+    "e2e": "tsdx test --config jest-e2e-config.json",
```

Comment:
we should converge on a standard for defining `e2e` tests within this repo. affiliates & core has `test-e2e`, API has `e2e`, and now the skd has `e2e`.

Following your pattern here, I'm ok with changing the others to all use `e2e` as you've defined here.

Reply:
hmm ok yea. i kept intuitively typing `yarn e2e`, which is why i changed it. we can discuss in eng meeting if anyone has any particular opinions on this. i will change it back for now